util/dvsim/Modes.py

```diff
@@ -266,6 +266,7 @@ def __init__(self, bdict):
         self.post_build_cmds = []
         self.en_build_modes = []
         self.build_opts = []
+        self.timeout_mins = 60
```

Comment:
```suggestion
        self.build_timeout_mins = 60
```

Should we uniquify this between builds and runs? Otherwise, this statement in dvsim:
```
  // In foo_sim_cfg.hjson
  timeout_mins: 30
```
will end up getting applied to build modes as well as the run modes (and test specifications). You are uniquifying this for the command-line override, but not in the hjson spec, which is a little odd. 

If you agree, you will need to make this change in the `Deploy` class derivatives as well.

Reply:
Then we end up having to complicate the launchers, since they don't want to know if it is a build or a run (or anything else): they just grab self.deploy.timeout_mins. If the timeouts are different variables that logic will need to change. I suppose we can create a Deploy get_timeout_mins function and specialize it for different derived Deploys. I'll try that.

Reply:
Yeah, that sounds good.